db/compaction/compaction_picker_test.cc

```diff
@@ -1342,13 +1342,142 @@ TEST_F(CompactionPickerTest, CompactionPriRoundRobin) {
             cf_name_, mutable_cf_options_, mutable_db_options_, vstorage_.get(),
             &log_buffer_));
     ASSERT_TRUE(compaction.get() != nullptr);
+    // Since the max bytes for level 2 is 120M, picking one file to compact
+    // makes the post-compaction level size less than 120M, there is exactly one
+    // file picked for round-robin compaction
     ASSERT_EQ(1U, compaction->num_input_files(0));
     ASSERT_EQ(selected_files[i], compaction->input(0, 0)->fd.GetNumber());
     // release the version storage
     DeleteVersionStorage();
   }
 }
 
+TEST_F(CompactionPickerTest, CompactionPriMultipleFilesRoundRobin1) {
```

Comment:
I am getting the impression from reading the test comments and running debugger that none of these newly added tests exercises the following logics https://github.com/facebook/rocksdb/pull/10341/files#diff-6862dde35f8188f1c00a45681b295230026a98e313f455f8d1a67a4d792ff725R368, https://github.com/facebook/rocksdb/pull/10341/files#diff-6862dde35f8188f1c00a45681b295230026a98e313f455f8d1a67a4d792ff725R400, https://github.com/facebook/rocksdb/pull/10341/files#diff-6862dde35f8188f1c00a45681b295230026a98e313f455f8d1a67a4d792ff725R415

Am I just wrong?

Reply:
Sure, I'm working on it now.